packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -4248,6 +4248,101 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /eservices/{eserviceId}/descriptors/{descriptorId}/reject:
+    post:
+      tags:
+        - eservices
+      summary: reject a delegated new e-service version
+      operationId: rejectDelegatedEServiceDescriptor
+      description: Rejects a delegated E-Service descriptor
```

Consistency with the rest of the M2M spec
```suggestion
      summary: Reject a delegated E-Service Descriptor in Waiting for Approval state
      operationId: rejectDelegatedEServiceDescriptor
      description: Reject a delegated E-Service Descriptor that is in Waiting for Approval state
```